mypy/dmypy_server.py

```diff
@@ -720,6 +741,9 @@ def pretty_messages(self, messages: List[str], n_sources: int,
 
     def update_sources(self, sources: List[BuildSource]) -> None:
         paths = [source.path for source in sources if source.path is not None]
+        if self.following_imports():
```

Comment:
I don't follow this really?

Reply:
If there are directories in paths, we'll get exceptions inside `fswatcher`.